Cython/Utility/CommonStructures.c

```diff
@@ -7,98 +7,90 @@ static PyObject* __Pyx_FetchCommonTypeFromSpec(PyType_Spec *spec, PyObject *base
 
 /////////////// FetchCommonType ///////////////
 
-static PyTypeObject* __Pyx_FetchCommonType(PyTypeObject* type) {
-    PyObject* fake_module;
-    PyTypeObject* cached_type = NULL;
-
-    fake_module = PyImport_AddModule((char*) "_cython_" CYTHON_ABI);
+static PyObject *fetch_fake_module(void) {
+    PyObject *fake_module = PyImport_AddModule((char*) "_cython_" CYTHON_ABI);
     if (!fake_module) return NULL;
     Py_INCREF(fake_module);
+    return fake_module;
+}
+
+static void verify_cached_type(PyObject *cached_type,
```

Comment:
Global C names must always be `__[Pp]yx_…` escaped to prevent naming collisions with user code.

Reply:
Done